integration-cli/docker_cli_history_test.go

```diff
@@ -82,3 +83,52 @@ func TestHistoryNonExistentImage(t *testing.T) {
 	}
 	logDone("history - history on non-existent image must fail")
 }
+
+func TestHistoryImageWithComment(t *testing.T) {
+
+	// make a image through docker commit <container id> [ -m messages ]
+	runCmd := exec.Command(dockerBinary, "run", "-i", "-a", "stdin", "busybox", "echo", "foo")
```

Comment:
I tried this command line, but it failed the test. I'd like to keep existing command line.

        --- FAIL: TestHistoryImageWithComment (0.35 seconds)
                docker_cli_history_test.go:100: error thrown while waiting for container: , exit status 1